### PR TITLE
[NOSTORY] Add support for RasFlashAlertBundle

### PR DESCRIPTION
**Pull request by johnmadrak**

### CODE REVIEW

File: Controller/AbstractCoreCRUDController.php [hunk 1]
```diff
@@ -86,12 +90,18 @@ public function renderCreate(Request $request)
...
         $form->handleRequest($request);
 
-        if ($form->isSubmitted() && $form->isValid()) {
-            $this->entityManager->persist($entity);
-            $this->entityManager->flush();
-            $routeInfo = $this->getCreateRouteRedirect($request, $entity);
+        if ($form->isSubmitted()) {
+            if ($form->isValid()) {
+                $this->entityManager->persist($entity);
+                $this->entityManager->flush();
```
Comment (luiztiago) [inline Controller/AbstractCoreCRUDController.php]: what do you think to use `persistAndFlush()` here?

File: Controller/AbstractCoreCRUDController.php [hunk 2]
```diff
@@ -147,11 +157,17 @@ public function renderEdit(Request $request, array $criteria)
         $editForm = $this->createForm($this->entityFormType, $entity);
         $editForm->handleRequest($request);
 
-        if ($editForm->isSubmitted() && $editForm->isValid()) {
-            $this->entityManager->persist($entity);
-            $this->entityManager->flush();
+        if ($editForm->isSubmitted()) {
+            if ($editForm->isValid()) {
+                $this->entityManager->persist($entity);
+                $this->entityManager->flush();
```
Comment (luiztiago) [inline Controller/AbstractCoreCRUDController.php]: same here

File: Controller/AbstractCoreCRUDController.php (hunk 1 above)
Comment (johnmadrak) [inline Controller/AbstractCoreCRUDController.php]: @luiztiago - ```persistAndFlush()``` is a custom method for our projects, it isn't a default method.

Comment (luiztiago) [inline Controller/AbstractCoreCRUDController.php]: =X

**Comment by luiztiago** [comment]
:shipit:

**Comment by johnmadrak** [comment]
Thanks @luiztiago!

**Pull request merged**